components/org.wso2.carbon.business.rules.templates.editor.core/src/main/react/components/Property.jsx

```diff
@@ -63,27 +64,20 @@ class Property extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            isExpanded: false,
+            isExpanded: false
```

Comment:
Need to have the ending command in JS objects to comply with Lint rules.

Reply:
No Lint rules have been added, hence no warning is given